assets/src/edit-story/components/keyboard/index.js

```diff
@@ -187,3 +187,20 @@ function isEditableTarget({ tagName, isContentEditable, type, readOnly }) {
   }
   return false;
 }
+
+export function prettifytShortcut(shortcut) {
```

Comment:
```suggestion
export function prettifyShortcut(shortcut) {
```

Also wondering if we should use `<kbd>` for displaying the shortcuts. Seems to be only semantic though 🤔